examples/todo/src/controllers/index.ts

```diff
@@ -1,6 +1,5 @@
-// Copyright IBM Corp. 2017,2018. All Rights Reserved.
-// Node module: @loopback/example-todo
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
```

Comment:
Shouldn't the example files preserve license headers? (I understand the files generated by `lb4` do not come with license headers.)